src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2809,6 +2813,8 @@ private UnboundLambda GetUnboundLambda(BoundLambda expr, VariableState variableS
         private MethodSymbol InferMethodTypeArguments(BoundCall node, MethodSymbol method, ImmutableArray<BoundExpression> arguments)
         {
             Debug.Assert(method.IsGenericMethod);
+            Debug.Assert(arguments.All(a => a.GetTypeAndNullability().NullableAnnotation.IsSpeakable()));
```

Comment:
>GetTypeAndNullability [](start = 46, length = 21)

`GetTypeAndNullability()` should be removed. Please use `GetNullableAnnotation()` instead.